Extension/src/LanguageServer/client.ts

```diff
@@ -2264,13 +2269,13 @@ export class DefaultClient implements Client {
      * order they were inserted.
      */
     private static async dispatch() {
-        // ensure that this is OK to start working
-        await this.isStarted;
-
         // reset the promise for the dispatcher
         DefaultClient.dispatching.reset();
 
         do {
+            // ensure that this is OK to start working
```

Comment:
It turns out that the dispatching can have stuff going when something crashes, so we move the check to inside the loop